### PR TITLE
Invalid remote report in dialog box

### PR DESCRIPTION
**Pull request by hakimdiallo**
The Invalid remote #106 report is now shown in a dialog box with the option to send the error report by mail to @maks.

### CODE REVIEW

File: src/main/res/values/strings.xml [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <string name="default_text">Default Text</string>
     <string name="title_clone_repo">Clone Repository</string>
     <string name="preference_file_key" translatable="false">com.manichord.mgit</string>
+    <string name="report_mail">maks@manichord.com</string>
```
Comment (maks) [inline src/main/res/values/strings.xml]: Could you please change this to mgit@manichord.com

**Comment by maks** [comment]
Thank you very much for contributing this! 
Just one small issue with the email address, if you could change that please.

**Comment by hakimdiallo** [comment]
Your welcome.
I changed it. Did u check it?

**Comment by maks** [comment]
@hakimdiallo sorry about that - I missed that you had updated it. Thanks again! This will go into the next release which I'm planning to get onto Play in the next few days.

**Pull request merged**